certbot/tests/cli_test.py

```diff
@@ -1,4 +1,7 @@
 """Tests for certbot.cli."""
+# Many tests in this file should be moved
+# into main_test.py and renewal_test.py.
```

Comment:
It's #2716, but I should reference that here. I'll send a follow up commit.

Nice catch though. In general, I don't like all the `TODO` and `XXX` comments in our code base. I think if an actual issue is made, it's much more likely to get done.

Also, for context, what happened is once upon a time, `cli.py` was an absolutely massive file. That file was split into `cli.py,` `main.py`, and `renewal.py`, but the tests weren't split into separate files as well.